integration/mediation-tests/tests-service/src/test/java/org/wso2/carbon/esb/endpoint/test/AddressEndpointTestCase.java

```diff
@@ -117,19 +71,10 @@ public void testSendingToInvalidAddressEndpoint()
 
     @Test(groups = { "wso2.esb" }, description = "Sending a Message to a Address endpoint has a Invalid Property")
     public void testSendingToInvalidPropertyAddressEndpoint()
-            throws XMLStreamException, FileNotFoundException, AxisFault, XPathExpressionException {
-        ProxyServiceAdminClient proxyAdmin = new ProxyServiceAdminClient(context.getContextUrls().getBackEndUrl(),
-                getSessionCookie());
-        ESBTestCaseUtils testUtil = new ESBTestCaseUtils();
-        try {
-            proxyAdmin.addProxyService(testUtil.loadResource(
-                    "/artifacts/ESB/endpoint/addressEndpointConfig/invalidPropertyAddressEndPoint.xml"));
-            Assert.fail("Proxy Deployment must failed due to Unsupported scope. but proxy deployment success");
-        } catch (Exception e) {
-            Assert.assertEquals(e.getCause().getMessage(),
-                    "Only 'axis2' or 'transport' or 'axis2-client' values are allowed for attribute scope for a property, Unsupported scope andun");
-        }
+            throws IOException {
 
+        boolean isDeployed = checkEndpointExistence(ENDPOINT_NAME);
```

Comment:
Do we need the check for existence as we have already added that endpoint to the server?